themes/Monokai++.tmTheme

```diff
@@ -278,14 +278,26 @@
 			<key>name</key>
 			<string>Tags</string>
 			<key>scope</key>
-			<string>entity.name.tag, entity.other.attribute-name</string>
+			<string>entity.name.tag</string>
 			<key>settings</key>
 			<dict>
 				<key>foreground</key>
 				<string>#f92672</string>
 			</dict>
 		</dict>
 
+		<dict>
+			<key>name</key>
+			<string>Tags</string>
+			<key>scope</key>
+			<string>entity.other.attribute-name</string>
+			<key>settings</key>
+			<dict>
+				<key>foreground</key>
+				<string>#fd971f</string>
+			</dict>
+		</dict>
+
```

Comment:
I'd prefer to move the `entity.other.attribute-name` scope to the `Member variables` dict at [line 245](https://github.com/dcasella/monokai-plusplus/blob/master/themes/Monokai%2B%2B.tmTheme#L245), I think it makes sense that attributes can be seen as parameters (like the `variable.parameter` scope)

Reply:
Yes, that makes sense. I will update the PR (thought that would be easier from within this thread, unless I'm missing the obvious button). Note that I think the Vue syntax highlighter is supposed to differentiate between at least 4 types of attribute in a Vue file:

```
<my-component
  bool-attrib
  attrib-with-value="42"
  :bound-attrib="variable"
  @click="methodName"
/>
```

but I'm not seeing the different scopes at the moment. Once that's resolved, this might need revisiting to highlight handlers (`@xxx`) as functions, etc.